### PR TITLE
Improve examples

### PR DESCRIPTION
**Pull request by qbarthelemy**
This PR:

- lowercases folders and scripts of examples;
- simplifies class `Connectivities`;
- sort imports, and replace ' by ".

### CODE REVIEW

File: examples/simulated/plot_toy_classification.py [hunk 1]
```diff
@@ -52,20 +52,20 @@
...
+                                          exponent=1, metric="riemann",
                                           return_num_denom=False)
         all_class_dis.append(class_dis)
 
     # average class distinctiveness across splits
     mean_class_dis = np.mean(all_class_dis)
     class_dis_array.append(mean_class_dis)
 
-    # Now let's train a MDM classifier and measure its performance
+    # Now let"s train a MDM classifier and measure its performance
```
Comment (gcattan) [inline examples/simulated/plot_toy_classification.py:63]: let's

File: examples/simulated/plot_toy_classification.py [hunk 2]
```diff
@@ -74,15 +74,15 @@
...
-ax2.set_xlabel(r'$\Delta/\sigma$', fontsize=14)
-ax1.set_ylabel(r'ROC AUC score', fontsize=12)
-ax2.set_ylabel(r'class distinctiveness', fontsize=12)
-ax1.set_title('Classification score and class distinctiveness value\n'
-              r'vs. class separability ($n_{dim} = 4$)',
+ax2.set_xlabel(r"$\Delta/\sigma$", fontsize=14)
+ax1.set_ylabel(r"ROC AUC score", fontsize=12)
+ax2.set_ylabel(r"class distinctiveness", fontsize=12)
+ax1.set_title("Classification score and class distinctiveness value\n"
+              r"vs. class separability ($n_{dim} = 4$)",
```
Comment (gcattan) [inline examples/simulated/plot_toy_classification.py:85]: r'vs

File: examples/simulated/plot_toy_classification.py (hunk 1 above)
Comment (qbarthelemy) [reply examples/simulated/plot_toy_classification.py:63]: Good catch!

**Review by gcattan — APPROVED** [review]
Good job @qbarthelemy ! That is some work there :)

**Pull request merged**